app/src/main/res/values/strings.xml

```diff
@@ -3,7 +3,7 @@
     <string name="org_name">AeroGear</string>
     <string name="org_url">www.aerogear.org</string>
     <string name="app_name">AeroGear Android Showcase</string>
-    <string name="app_description">A Native Android Application that demonstrates features of the Aerogear Android SDK.</string>
+    <string name="app_description">This app demonstrates how to use the AeroGear Mobile services, backed by OpenShift to perform a wide range of common mobile tasks such as Identity Management, Device Security, Metrics Capture, Push Notifications and more. For further information, visit www.aerogear.org</string>
```

Comment:
can we make the link clickable? It should open the phone browser and direct user to the website.